Deep-Drop/core/deepdrop.py

```diff
@@ -33,30 +32,26 @@ def parse_process_list(process_list):
 
     processes = namedtuple('process', 'pid, owner, name')
 
+    hostname = ''
```

Comment:
Good catch. This code came from parsing lots of process lists. 

will return False on exception.